backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT23.json

```diff
@@ -12,6 +12,6 @@
     "Hostname": "at23.altinn.cloud"
   },
   "KeyVaultSettings": {
-    "SecretUri": "https://altinn-at23-am-kv.vault.azure.net/"
+    "SecretUri": "https://altinn-amui-at23-kv.vault.azure.net/"
```

Comment:
Isn't this wrong in the other env-appsettings files then?

Reply:
Ahh, it's just wrong in AT23 since it's still running on the temporary setup correct?

Reply:
Right :)